templates/template.sql

```diff
@@ -1,5 +1,6 @@
 PRAGMA foreign_keys=OFF;
 BEGIN TRANSACTION;
+DROP TABLE IF EXISTS col;
```

Comment:
Let's try to fix it then instead of editing `SQL`. That changes tell me that, now, somewhere same `db` instance used twice.